vuepress/docs/docs/README.md

```diff
@@ -93,9 +93,7 @@ Use `ResponseValue#toString` to get the JSON representation of the result.
 ```scala
 val query = """
   {
-    characters {
-      name
-    }
+    characters 
```

Comment:
Why this change? This wouldn't be valid GraphQL.

Reply:
Because I didn't understand what I was doing. Sorry. I'll pay more attention next time.